test/unit/reducers/projects.js

```diff
@@ -164,6 +172,30 @@ tap(initProjects({1: false}), projects =>
   )),
 );
 
+tap(initProjects({1: false}), projects =>
+  test('hideComponent', reducerTest(
+    reducer,
+    projects,
+    partial(hideComponent, '1', 'output', now),
+    projects.update('1', projectIn =>
+      projectIn.set('hiddenUIComponents', new Immutable.Set(['output'])).
+        set('updatedAt', now),
+    ),
+  )),
+);
+
+tap(initProjects({1: true}), projects =>
+  test('unhideComponent', reducerTest(
+    reducer,
+    projects.update('1', projectIn =>
+      projectIn.set('hiddenUIComponents', new Immutable.Set(['output'])).
+        set('updatedAt', now),
```

Comment:
@outoftime I reverted the changes to `initProjects` and updated this test. One thing I noticed is that this second `set` operation is redundant since passing the `modified` flag sets the `updatedAt` property. So maybe it's fine to omit it. And for the new test above this one, I could use `initProjects({1: true})` and do the same. Thoughts?